OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum ReferencePoints { TopLeft, BottomRight }
```

Comment:
As far as the coding is concerned, not at all. Even stuff like `Center`, `TopCenter`, `BottomCenter`, `CenterLeft` and `CenterRigh` crossed my mind. The only question would be the usefulness of such a thing. Do you have ideas of use cases?